src/screens/application/ApplicationStatus.tsx

```diff
@@ -1,46 +1,21 @@
 import React from "react";
-import { Box, Stack, VStack, HStack, Text } from "@chakra-ui/react";
+import { Box, Stack } from "@chakra-ui/react";
 import "../../assets/styles/App.css";
 import Layout from "../../components/common/layout/Layout";
-import { useNavigate } from "react-router-dom";
-import { CheckIcon } from "@chakra-ui/icons";
+import ApplicationList from "../../components/ApplicationList";
+import { applicationList } from "../../assets/mockdata/applicationList";
 
 const MyApplications: React.FC = () => {
-  const navigate = useNavigate();
-  const handleBack = () => {
-    navigate(-1);
-  };
-
   return (
     <Layout
       _heading={{
-        heading: "Pre-matric Scholarship-ST",
-        handleBack,
+        heading: "My Applications",
```

Comment:
use lang constants

Reply:
I haven't installed i18n yet. After Sonali's PR changes are merged, I will update it.